### PR TITLE
Refactor/inverted covered handling 2

### PR DESCRIPTION
**Pull request by Gymnasiast**
This is another chapter in normalising track drawing, following a plan @IntelOrca and I made.

 Specifically, it does the following:
- Group available pieces per track style and inverted/non-inverted status
- Gets rid of the alternative RTDs for the Flying, Multi-dim and Lay-down coasters
- Make it possible to use different track drawers for inverted/non-inverted track, bringing them much more in line with other coaster types

When the work this PR is part of is finished, stuff like this should be possible:
- Multiple track styles for a ride type (beyond just a handful of covered pieces)
- Making the RTDs into objects
- (In the longer run) defining custom drawing code for new track styles.

Needs testing, obviously. The construction window will be the area of interest here.

### CODE REVIEW

**Review by karst — APPROVED** [review]
Everything seems to be in working order :)

**Review by tupaschoal — APPROVED** [review]
A bit hard to follow, but it seems to be ok. Haven't tested
